cabby/evals/eval_geo_model.py

```diff
@@ -0,0 +1,120 @@
+# coding=utf-8
+# Copyright 2020 Google LLC
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Utility for distance based evaluation for geo models in RVS.
+
+Example:
+$ bazel run -- cabby/evals/eval_geo_model \
+        --input_file /mnt/hackney/data/cabby/evals/sample_predictions.tsv
+The input_file is expected to be a tab-separated file with:
+        example_id
+        true_latitude
+        true_longitude
+        predicted_latitude
+        predicted_longitude
+Any lines which do not comply to this format will be skipped.
+"""
+
+import collections
+
+import numpy as np
+from absl import app, flags
+from geopy.distance import great_circle
+
+from cabby import logger
+
+FLAGS = flags.FLAGS
+flags.DEFINE_string("input_file", None,
+                    "TSV file containing true and predicted co-ordinates.")
```

Comment:
The model outputs the prediction and ground truth in a 'pt' format so a converstion to an 'tsv' is needed. Maybe add a function that does that?